README.adoc

```diff
@@ -12,3 +12,38 @@ The `xorg.conf` file should be placed in `/etc/X11/`.
 The `config.txt` file is the equivalent of a BIOS on the pi. It contains system configuration parameters. 
 This file is needed on the primary and secondary pi.
 The `config.txt` file should be placed in `/boot/`.
+
+= Instructions to run SetupScript
+
+Ensure to clone Repo to home directory (~) in order for the script to know where to move patch files upon running of the script.
+----
+cd ~
+git clone https://github.com/UCSolarCarTeam/Epsilon-Raspberry.git
+----
+Navigate to Epsilon-Raspberry Directory.
+----
+cd Epsilon-Raspberry
+----
+Before running the setup script, make sure that:
+
+* You have consistent, stable internet connection.
+* You won't need to use the pi or move your computer until the script is complete.
+The script may take several *hours* to run, so be prepared to leave your computer on all night.
+----
+./SetupScript
+----
+Flash forward to after completion, CONGRATULATIONS you are almost there, I believe in you.
+Navigate to your .bashrc file in your home (~/) directory and add the following two lines:
```

Comment:
Most companies won't accept this kind of tone. For the sake of making people industry ready, let's avoid stuff like this :) 

Another important thing to note is that professional documentation is also always in third person. So 
```
Navigate to your .bashrc file in your home (~/) directory and add the following two lines:
```
Should be
```
Add the following two lines to the `~/.bashrc` file.
```
  Do this for the rest of the README